pkgs/by-name/wl/wl-clipboard-rs/package.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, wayland
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "wl-clipboard-rs";
+  version = "unstable-2023-11-27";
```

Packaging the unstable currently because the latest release `v0.8.0` does not have a `Cargo.lock` file yet.

---

https://github.com/NixOS/nixpkgs/tree/c34f9cf3abfb5183a43c2a0a86339af41cd2f391/pkgs#package-naming

> - The version attribute must start with a digit e.g`"0.3.1rc2".
> 
> - If a package is a commit from a repository without a version assigned, then the version attribute should be the latest upstream version preceding that commit, followed by -unstable- and the date of the (fetched) commit. The date must be in "YYYY-MM-DD" format.

```suggestion
  version = "0.8.0-unstable-2023-11-27";
```

---

Done.